00_Base/src/model/ServerCredentialsRole.ts

```diff
@@ -67,3 +82,18 @@ export class ServerCredentialsRole extends Model implements ICredentialsRole {
     return serverCredentialsRole;
   }
 }
+
+export const toCredentialsRoleDTOFromServerCredentialsRole = (
```

Comment:
Why not make this method a part of `ServerCredentialsRole` class?

Then you can make call: `serverCredentialsRole.toCredentialsRoleDto()`.

Reply:
agreed - but also please make it `static`.

Please note that there may be issues if your class extends Model and you are trying to define a `static build(..)` method because of conflicts. In the majority of other cases where `static` can be used, we should. In some cases, static context is limited for... reasons... and this approach can be used as an alternative. In some cases, you may want to move a `const fun() => {}` into a util or something so that it can be imported and reused without having the import the rest of the class, especially if that class extends Model or has some other imports that you are trying to avoid.

I think this may also have been something that we had previously done for seeders because they didnt have the same Models imported so we would get a sequelize error - but now we saw that we can resolve that by making sure that `DefaultSequelizeInstance` or `OcpiSequelizeInstance` is initialized, whichever applicable.